pkg/manifests/config.go

```diff
@@ -136,11 +176,12 @@ type AlertmanagerMainConfig struct {
 }
 
 type ThanosRulerConfig struct {
-	LogLevel            string                               `json:"logLevel"`
-	NodeSelector        map[string]string                    `json:"nodeSelector"`
-	Tolerations         []v1.Toleration                      `json:"tolerations"`
-	Resources           *v1.ResourceRequirements             `json:"resources"`
-	VolumeClaimTemplate *monv1.EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate"`
+	LogLevel             string                               `json:"logLevel"`
+	NodeSelector         map[string]string                    `json:"nodeSelector"`
+	Tolerations          []v1.Toleration                      `json:"tolerations"`
+	Resources            *v1.ResourceRequirements             `json:"resources"`
+	VolumeClaimTemplate  *monv1.EmbeddedPersistentVolumeClaim `json:"volumeClaimTemplate"`
+	AlertManagersConfigs []AdditionalAlertmanagerConfig       `json:"additionalAlertManagerConfigs"`
```

Comment:
I noticed that this typo was also present in the precedent PR and IMO we should fix it since `AlertManager` is counterintuitive.
```suggestion
	AlertManagersConfigs []AdditionalAlertmanagerConfig       `json:"additionalAlertmanagerConfigs"`
```

Reply:
On second look, many of those typos are actually introduced in this PR. I added a new commit renaming `additionalAlertManagerConfigs` to `additionalAlertmanagerConfigs`